tests/runner.py

```diff
@@ -70,6 +72,11 @@ def is_emterpreter(self):
 
   def setUp(self):
     Settings.reset()
+
+    for root, dirnames, filenames in os.walk(TEMP_DIR):
```

Comment:
Do you have a better idea on how we could ensure testing that Emscripten operations do not leak temp files or directories? Currently we still have a potential for such leaks, especially when tests fail. This is only the test runner though, can you free up /tmp regularly to free up space and make sure that this function will run fast? :)